### DISCUSSION TITLE
Performance and Disk Space Issues in Destination Postgres v2.4.0

### DISCUSSION THREAD

**Discussion by LSalmada** [airbytehq/airbyte#48750] [Connector Questions]
Hi everyone,

I've noticed a significant decrease in performance when using version 2.4.0 of the destination-postgres connector compared to version 0.4.0. The synchronization process seems to take much longer, and I'm trying to understand the reasons behind this change.
I'm syncing data between BigQuery and Postgres, my postgres instance is hosted on CrunchyBridge(gcp)

![image](https://github.com/user-attachments/assets/e7d3e0a1-0a8e-4d3d-b8a9-5513e39cf30e)

Additionally, I’m experiencing high disk usage during synchronization. Is there a recommended way to optimize disk space consumption in this version?

Any insights into the performance differences or tips for reducing disk usage would be greatly appreciated.

Thanks in advance!

**Comment by luizmiranda-engine** [comment]
Hey @LSalmada have you found something about the slowness? We are facing the same issue with our tables.
